.env

```diff
@@ -0,0 +1 @@
+MONGODB_URI=mongodb://admin:iulilguzfg867foliO98f@195.201.170.57:20000/chrat?authSource=admin
```

Comment:
The .env file should be in your gitignore file and not pushed to github.

Reply:
jeah but i cant use env variable at vercel i want to try the merge and function after merge at vercel